src/views/TeamView.tsx

```diff
@@ -93,18 +94,21 @@ export const TeamView = (props: Props): JSX.Element => {
     event.preventDefault();
     setInviteMessage("");
 
-    const result = await props.services.inviteUser({
-      email: inviteEmail,
-    });
-
-    if (result) {
-      setInviteEmail("");
-      setInviteMessage("Invite was sent");
-    } else {
-      setInviteMessage("An error happened with the invite");
+    try {
+      const result = await props.services.inviteUser({
+        email: inviteEmail,
```

Comment:
I think `inviteUser` should also be changed for this to work as it is? The method has its own try-catch..